.gitignore

```diff
@@ -8,7 +9,7 @@
 .vs/
 .vscode/
 .tools/
-/build/loc/Properties/launchSettings.json
+/eng/tools/OneLocBuildSetup/Properties/launchSettings.json
```

Comment:
This location wasn't updated properly from when I moved files around previously.